modules/data.atmosphere/R/met.date.process.R

```diff
@@ -0,0 +1,213 @@
+##' @title met.process.set.stage
+##' @export
+##' 
+##' @param input.info.list ,list containing DB record ids, start-dates and end_dates
+##' @param start_date , Start date as given by run
+##' @param end_date , End date as given bu run
+##' @param con, database connection
+##' @author Tony Gardella, Elizabeth Cowdery
```

Comment:
Documentation says nothing about what this function is supposed to do or what it returns.  Drop commas between param and description